tests/EvalTests.hs

```diff
@@ -417,26 +419,27 @@ genEvalCompareTests = do
     mkTestCase td f = testCase f $ assertEvalFileMatchesNix (td </> f)
 
 
-instance (Show r, Show (NValueF m r), Eq r) => Eq (NValueF m r) where
-    NVConstantF x == NVConstantF y = x == y
-    NVStrF ls == NVStrF rs = hackyStringIgnoreContext ls == hackyStringIgnoreContext rs
-    NVListF x == NVListF y = and (zipWith (==) x y)
-    NVSetF x _ == NVSetF y _ =
-        M.keys x == M.keys y &&
-        and (zipWith (==) (M.elems x) (M.elems y))
-    NVPathF x == NVPathF y = x == y
-    x == y = error $ "Need to add comparison for values: "
-                 ++ show x ++ " == " ++ show y
+-- instance (Show r, Show (NValueF p m r), Eq r) => Eq (NValueF p m r) where
```

Comment:
Why is this commented out?

Reply:
It wasn't needed anymore. Should be deleted.